blueocean-dashboard/src/main/less/mixins/text-overflow.less

```diff
@@ -1,5 +1,6 @@
 // Text overflow
 // Requires inline-block or block for proper styling
+// Works with flexbox as well as long as the min-width property is set to something else than the default 100% that is set for flexbox
```

Comment:
In retrospect, I should have named this differently, perhaps `overflow-ellipsis`. I can't insist you rename something that I myself named poorly, but if you want to obsess over names like I do, go ahead and change it :)